changelogs/unreleased/gh-3211-per-module-log_level.md

```diff
@@ -0,0 +1,7 @@
+## feature/box
```

Comment:
Nit: Should be feature/core, because this feature works even if the box isn't configured.

Reply:
Fixed.